test/generator/html-elements/output.js

```diff
@@ -0,0 +1,115 @@
+Foo.propTypes = {
+	bothTypes: PropTypes.oneOfType([
```

Comment:
Maybe it would be better if we could replace this code with a custom element prop-types hosted in `@material-ui/core/utils`.

Reply:
With https://github.com/merceyz/typescript-to-proptypes/pull/10 that is an option